app/static/tests/unit/components/SessionInitialiseModal.test.ts

```diff
@@ -0,0 +1,39 @@
+import { shallowMount } from "@vue/test-utils";
+import SessionInitialiseModal from "../../../src/app/components/SessionInitialiseModal.vue";
+
+describe("SessionInitialiseModal", () => {
+    const getWrapper = (open = true) => {
+        return shallowMount(SessionInitialiseModal, { props: { open } });
+    };
+
+    it("renders as expected when open is true", () => {
+        const wrapper = getWrapper();
+        expect(wrapper.find("#session-initialise-modal .modal").classes()).toContain("show");
+        expect((wrapper.find("#session-initialise-modal .modal").element as HTMLDivElement).style.display)
+            .toBe("block");
+        expect(wrapper.find(".modal-backdrop").exists()).toBe(true);
+        expect(wrapper.find(".modal-body").text())
+            .toBe("Would you like to reload the most recent session or start a new session?");
+        expect(wrapper.find("button#reload-session").text()).toBe("Reload session");
+        expect(wrapper.find("button#new-session").text()).toBe("New session");
+    });
+
+    it("renders as expected when open is false", () => {
+        const wrapper = getWrapper(false);
+        expect(wrapper.find("#session-initialise-modal .modal").classes()).not.toContain("show");
```

Comment:
also any nicer selectors? as the user can not see ids, and class names we should avoid using them for testing purposes unless absolutely necessary.